GPT/staticPrompt.talon-list

```diff
@@ -34,11 +33,9 @@ convert to yam: Convert the following data into a yaml format.
 
 ## ACCESSIBILITY
 describe code: explain what the following code does in natural language at a high level without getting into the specifics of the syntax.
-describe structure: describe the structure of the following text. Do not describe the content. Describe the structure of the text.
-describe summary: Condense the following text into a summary that is less verbose but still gives me the gist of the content. Do not describe the formatting.
-check grammar: Check the grammar of the following text. Return a list of all potential errors.
+check grammar: Check the grammar and formatting of the following text. Return a list of all potential errors.
 check spelling: Check the spelling of the following text. Return a list of all potential errors.
-check structure: I want you to skim the structure and layout of the following text. Tell me if the structure and order of my writing is correct. If it is not correct or flows poorly then tell me what might be wrong with it. If it is all correct then say it looks good. Do not describe the formatting. Do not comment on the specific content only its ordering and layout.
+check structure: Skim the structure and layout of the following text. Tell me if the structure and order of my writing is correct. If it is not correct or flows poorly then tell me what might be wrong with it. If it is all correct then say it looks good. Do not describe the formatting.
```

Comment:
I read somewhere that negative assertions like "do not describe" do not work as well as positive ones like "describe only the structure"

Reply:
Yup so isn't this fine then? I removed that negative assertion

Reply:
I think there's still a negative assertion here: 'Do not describe the formatting.'

Reply:
fixed